sql/updates/2.20.2--2.20.3.sql

```diff
@@ -0,0 +1,5 @@
+-- Make chunk_id use NULL to mark special entries instead of 0
```

Comment:
When forward porting this change, it is also necessary to remove the corresponding SQL code from the development update script in reverse-dev.sql and latest-dev.sql.

This is probably why the update and downgrade tests are failing.

Reply:
YES! That's right 🤦  Thank you Erik for your diligence. I fixed that via https://github.com/timescale/timescaledb/pull/8249/commits/5b7594b1921c05bee558dd18eb984c6fc98e8a5d